src/stickyfill.js

```diff
@@ -317,6 +320,12 @@ const Stickyfill = {
     stickies,
     Sticky,
 
+    forceSticky () {
+        seppuku = false;
+        init();
```

Comment:
We need a check to prevent the `init` function from running twice.  `isInitialized` flag or something.